components/omega/test/timeStepping/TimeStepperTest.cpp

```diff
@@ -0,0 +1,314 @@
+#include "TimeStepper.h"
+#include "../ocn/OceanTestCommon.h"
+#include "AuxiliaryState.h"
+#include "DataTypes.h"
+#include "Decomp.h"
+#include "Halo.h"
+#include "HorzMesh.h"
+#include "IO.h"
+#include "IOField.h"
+#include "Logging.h"
+#include "MachEnv.h"
+#include "MetaData.h"
+#include "OceanState.h"
+#include "OmegaKokkos.h"
+#include "TendencyTerms.h"
+#include "mpi.h"
+
+#include <cmath>
+#include <iomanip>
+#include <iostream>
+
+using namespace OMEGA;
+
+constexpr Geometry Geom   = Geometry::Planar;
+constexpr int NVertLevels = 1;
+
+struct DecayThicknessTendency {
+   void operator()(Array2DReal ThicknessTend, const OceanState *State,
+                   const AuxiliaryState *AuxState, int ThickTimeLevel,
+                   int VelTimeLevel, Real Time) const {}
+};
+
+struct DecayVelocityTendency {
+   Real Coeff = 0.5;
```

Comment:
I was wondering if it would be easy to import some constants defined in a YAML file here. For example, the wavenumber of the manufactured solution is defined in the YAML file and we need to include it here. I think it should be easy, but I thought it would be good to check.

Reply:
It should be straightforward. Just make the constructor of this class take in `Config`.